### PR TITLE
Adds mounting point support

### LINKED ISSUE #66: Fix MountingPoint in Components
## Description
Address the issue with the improper functioning of mountingPoint in "components".

## Task
- Fix the issue with mountingPoint in "components"

### PR DESCRIPTION
**Pull request by PascalSenn**

### CODE REVIEW

File: src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs [hunk 1]
```diff
@@ -59,27 +61,104 @@ public JsonSchema Compose(IEnumerable<Component> components, IEnumerable<Variabl
...
+            case 1:
+                schema = Schemas[0];
+                break;
+
+            default:
+                schema = new JsonSchemaBuilder();
+                break;
+        }
+
+        if (Schemas.Count > 0)
```
Comment (RohrerF) [inline src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs]: remove

File: src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs
Comment (RohrerF) [inline src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs:1]: Please add tests for this new behaviour

**Review by RohrerF — CHANGES_REQUESTED** [review]

File: src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs [hunk 2]
```diff
@@ -59,27 +61,104 @@ public JsonSchema Compose(IEnumerable<Component> components, IEnumerable<Variabl
...
+        {
+        }
```
Comment (PascalSenn) [inline src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs]:
```suggestion
```

File: src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs [hunk 3]
```diff
@@ -59,27 +61,101 @@ public JsonSchema Compose(IEnumerable<Component> components, IEnumerable<Variabl
...
+
+            case 1:
+                schema = Schemas[0];
+                break;
+
+            default:
+                schema = new JsonSchemaBuilder();
+                break;
+        }
+
```
Comment (PascalSenn) [inline src/Confix.Tool/src/Confix.Tool/Entities/Project/ProjectComposer.cs:116]:
```suggestion
```

**Pull request merged**